File: VerUpdater/trunk/source/web/VerUpdaterSvr/common/src/main/java/com/ebeijia/common/AjaxResp.java
package com.ebeijia.common;

import java.util.HashMap;
import java.util.Map;

import net.sf.json.JSONObject;

import com.ebeijia.contstant.RespCode;
import com.ebeijia.util.DateTime4J;

public class AjaxResp {
	public static Map<String, Object> getReturn(String respCode,Object o) {
		String sendTime=DateTime4J.getCurrentDateTime();
		Map<String,Object> map= new HashMap<String,Object>();
		    map.put("sendTime", sendTime);
		    map.put("rspCode", respCode);
		    map.put("rspMsg", RespCode.Msg(respCode));
		    map.put("data", o);
		   //String sign=EndResp.resp(sendTime, o);
		    map.put("signType", "01");
		   // map.put("sign", sign);
		    return map;
	}
	public  Map<String, Object> getReturn(String respCode,Object o,String msg){
		String sendTime=DateTime4J.getCurrentDateTime();
		Map<String,Object> map= new HashMap<String,Object>();
		 	map.put("sendTime", sendTime);   
		 	map.put("rspCode", respCode);
		    map.put("rspMsg", msg);
		    
		    map.put("content", o);
		   // String sign=EndResp.resp(sendTime, o);
		    map.put("signType", "01");
		    //map.put("sign", sign);
		    return map;
		  }
}	

File: VerUpdater/trunk/source/console/core/src/main/java/com/ebeijia/ajax/resp/AjaxResp.scala
package com.ebeijia.ajax.resp

import java.util.{HashMap, Map}

import com.ebeijia.util.core.RespCode

/**
 * AjaxResp
 * @author zhicheng.xu
 *         15/8/7
 */
object AjaxResp {
  def getReturn(respCode: String, o: AnyRef): Map[String, AnyRef] = {
    val map: Map[String, AnyRef] = new HashMap[String, AnyRef]
    map.put("respCode", respCode)
    map.put("errorInfo", RespCode.Msg(respCode))
    map.put("content", o)
    map
  }

  def getReturn(respCode: String, o: AnyRef,msg:String): Map[String, AnyRef] = {
    val map: Map[String, AnyRef] = new HashMap[String, AnyRef]
    map.put("respCode", respCode)
    map.put("errorInfo", msg)
    map.put("content", o)
    map
  }
}

